src/index.js

```diff
@@ -104,6 +104,7 @@ const ApolloClientPlugin = createPlugin({
         : [authMiddleware, connectionLink];
       const client = new ApolloClient({
         ssrMode: __NODE__ ? true : false,
+        connectToDevTools: (__BROWSER__ && __DEV__) ? true : false,
```

Comment:
I wonder if it should just be `__BROWSER__ && __DEV__`. It does look like other defines for this object use this style though, so maybe consistency is better. Thanks for the PR!

Reply:
@KevinGrandon So we should remove `( )` right ?

Reply:
Yup. I just made the commit because I saw eslint complained.